scripts/install/InitMetadataService.php

```diff
@@ -60,8 +60,8 @@ public function __invoke($params)
             }
         } else {
             $importerConfig = [
-                MetadataImporter::INJECTOR_KEY     => [],
-                MetadataImporter::EXTRACTOR_KEY    => [],
+                MetadataImporter::INJECTOR_KEY     => ['oat\\taoQtiItem\\model\\qti\\metadata\\ontology\\LomInjector'],
```

Comment:
In order to make these lines look a bit cleaner, we can use the `LomInjector::class` notation instead.